.github/workflows/pr-label.yml

```diff
@@ -12,6 +12,29 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+  labeler:
+    permissions:
+      issues: write
```

Comment:
Is this necessary?

Reply:
I'l;l try and remove it. Can you approve the workflow run and see if it works

Reply:
It might also need `contents: read`

Reply:
Seems to not like that

Reply:
probably going to make it its own file if that works

Reply:
Why so?

Reply:
To be honest, I'm tired of messing with pipelines for shirt sizing, so its a step in the right direction so I can start the coverage stuff

Reply:
Coverage is much more important anyways. And I'm sorry if it comes across as if I'm hassling you, that's not my intention of course. I'm just trying to take my responsibility to make sure I understand the changes going into the repo.

Reply:
I totally understand the responsibility required to keep code clean and quality. Getting those thresholds is paramount because they will directly influence the code coverage numbers so I’m waiting on them to do the coverage. We can also join them in this PR but I expect that’s not what is wanted